virttest/qemu_storage.py

```diff
@@ -151,6 +152,32 @@ def get_image_repr(image, params, root_dir, representation=None):
     return func(image, params, root_dir)
 
 
+def create_image_offline_snapshot(params, vm_name, image_name):
```

Comment:
Suggest the name `create_temporary_image_snapshots`

Reply:
Thanks, updated.